bin/spiced/src/lib.rs

```diff
@@ -376,7 +376,7 @@ fn apply_overrides(
                 }
                 .fail();
             }
-        };
```

Comment:
Many syntax changes came from the new rustc version and running 
```bash
cargo clippy --all-targets --allow-dirty --features aws-secrets-manager,keyring-secret-store,models,odbc,release --workspace --fix
```

Reply:
Are you running 1.86? If you pin to 1.85 these shouldn't pop up. I'm ok to fix these now, but it should come in a separate PR - not one that is tied to the Llama 4 upgrade.